src/amr/tagging/default_hybrid_tagger_strategy.h

```diff
@@ -0,0 +1,79 @@
+#ifndef DEFAULT_HYBRID_TAGGER_STRATEGY_H
+#define DEFAULT_HYBRID_TAGGER_STRATEGY_H
+
+#include "hybrid_tagger_strategy.h"
+#include "core/data/grid/gridlayoutdefs.h"
+#include "core/data/vecfield/vecfield_component.h"
+
+
+namespace PHARE::amr
+{
+template<typename HybridModel>
+class DefaultHybridTaggerStrategy : public HybridTaggerStrategy<HybridModel>
+{
+    using gridlayout_type           = typename HybridModel::gridlayout_type;
+    static auto constexpr dimension = HybridModel::dimension;
+
+public:
+    void tag(HybridModel& model, gridlayout_type const& layout, int* tags) const override;

```

Comment:
is this interacting with SAMRAI somehow that it has to be a raw "int"?

Reply:
yes